modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -452,6 +458,9 @@ const OrderableTable = ({
 				</ManagementToolbar.Container>
 
 				{items.length ? (
+
+					// @ts-ignore
```

Comment:
Anything we should consider here? Removing the comment does not throw any warning or error

Reply:
If the comment is removed, you get this type error

![Screenshot 2024-10-01 at 9 56 55 AM](https://github.com/user-attachments/assets/eea3160a-1e5f-4d1f-87d8-7e4974e49796)